examples/advanced/sklearn-svm/jobs/sklearn_svm_base/app/custom/svm_assembler.py

```diff
@@ -33,7 +32,8 @@ def __init__(self, kernel):
         self.support_y = None
         self.kernel = kernel
 
-    def get_model_params(self, data: dict):
+    def get_model_params(self, dxo: DXO):
+        data = dxo.data
         return {"support_x": data["support_x"], "support_y": data["support_y"]}
 
     def assemble(self, data: Dict[str, dict], fl_ctx: FLContext) -> dict:
```

Comment:
please update this method signature to

`def assemble(self, data: Dict[str, dict], fl_ctx: FLContext) -> DXO:`

for all child class and parent class in nvflare/app_common/aggregators/assembler.py

Thanks

Reply:
ah yes, thanks!